salt/addons/dex/main.sls

```diff
@@ -0,0 +1,13 @@
+include:
+  - .pre
+
+explicit-pre-dependency:
+  test.nop:
+  - require:
+      - sls: addons.dex.pre
```

Comment:
Thiss is not very elegant.. do we really need this? We are already setting the order in the orchestration...

Reply:
@inercia I also didn't like having to artificially splitting the states.

The problem I had with everything being in one file - the `copying` of the `15-secret.yaml` did not happen in that case - I *assumed* that is because the macro-expansion have to run first, which already run other states.

However I just tried moving everything into one file again and this time it worked ...